### ISSUE TITLE
restore api calls

### ISSUE THREAD

**Issue by alishaevn** [scientist-softserv/webstore#161]
# story
as a result of our updates on https://github.com/scientist-softserv/webstore/issues/155, no api calls are being returned. everything remains in "loading" status.

# acceptance criteria
- [x] api calls return their expected values again

# demo

https://user-images.githubusercontent.com/29032869/215134791-a721177a-6206-4da9-b6c5-fabf214b2cd0.mp4

**Comment by summer-cook** [comment]
@alishaevn 
I just pulled in main, and do not have the same issue
<img width="848" alt="image" src="https://user-images.githubusercontent.com/73361970/215142927-7a6c979f-57d1-49b9-8b77-d8740756d9f4.png">
<img width="780" alt="image" src="https://user-images.githubusercontent.com/73361970/215142979-471eb85d-9ea2-4408-a9d3-38d5193e18db.png">

**Comment by alishaevn** [comment]
^ summer && I continued the above convo in slack. 

tldr -- after removing her local access token and pulling the latest version of the wcl, we're having the same experience.

however, this may be a side effect of not having supplier auth set up yet. putting this on hold until #133 is completed.

**Comment by alishaevn** [comment]
#67 is completed, but each api call still remains in "loading" status for a logged out user. picking this work back up.
